anta/tests/interfaces.py

```diff
@@ -474,8 +502,8 @@ class Input(AntaTest.Input):
 
         mtu: int = 1500
         """Default MTU we should have configured on all non-excluded interfaces. Defaults to 1500."""
-        ignored_interfaces: list[str] = Field(default=["Management", "Loopback", "Vxlan", "Tunnel"])
-        """A list of L3 interfaces to ignore"""
+        ignored_interfaces: list[AllInterfacePrefix | Interface] = Field(default=["Management", "Loopback", "Vxlan", "Tunnel"])
```

Comment:
Here we may have L3Interfaces only.

```suggestion
        ignored_interfaces: list[AllInterfacePrefix | L3Interface] = Field(default=["Management", "Loopback", "Vxlan", "Tunnel"])
```
Or 

We should have two different flags: one to ignore all L2 interfaces (as we currently have), and a second one to ignore specific L3 interfaces.

Reply:
Similarly for L2 interfaces, we should only specify the L2 interfaces to ignore, and all other L3 interfaces should be ignored by default."

Reply:
@MaheshGSLAB

Reply:
We may have to find a way for vxlan as actually operates at both L2 and L3

Reply:
But we cant segregate the interface as L2 or L3 without checking the details on an interface so Interface is correct one here. 
For example an ethernet interface can be L2 and L3, we cant get it by just the name of interface.